lib/system/signal.go

```diff
@@ -16,5 +27,8 @@ import "C"
 // http://garethrees.org/2015/08/07/ping/
 // https://github.com/openssh/openssh-portable/commit/4e0f5e1ec9b6318ef251180dbca50eaa01f74536
 func ResetInterruptSignalHandler() {
-	C.resetInterruptSignalHandler()
+	result := C.resetInterruptSignalHandler()
+	if result != 0 {
```

Comment:
The value you got back was a `C.int` not a Go `int` so I'm not sure this comparison will do what you want. To be on the safe side, I'd do the following:

```
if int(result) != 0 {
   log.Warnf("Failed to reset interrupt signal handler: %v.", int(result))
}
```

We can parse errno at some other point to get a nice error message.